server/svix-server/config.default.toml

```diff
@@ -39,6 +39,13 @@ log_format = "default"
 # always sending. If the OpenTelemetry address is not set, this will do nothing.
 # opentelemetry_sample_ratio = 1.0
 
+# The Sentry DSN to use for error reporting. Disabled when omitted/null
+# sentry_dsn = "https://somedsn.ingest.sentry.io/12345"
+
+# The environment that the server is running in.
+# Supported: "dev", "staging", "prod"
+environment = "dev"
```

Comment:
Let's keep it unset here so that we are sure we set it correctly in terraform (and not forget by accident).